code/modules/mob/living/carbon/superior_animal/robots/robot.dm

```diff
@@ -128,21 +128,3 @@
 		var/robotics_expert = user.stats.getPerk(PERK_ROBOTICS_EXPERT)
 		if(robotics_expert) // Are we an expert in robots?
 			to_chat(user, SPAN_NOTICE("[name] is currently at [(health/maxHealth)*100]% integrity!")) // Give a more accurate reading.
-		else if(health < maxHealth * 0.10)
```

Comment:
why are you removing these?

Reply:
Wait nvm theirs a ..() at the top
this means its prob doble doing it

Reply:
to fix a bug where synthetics would have t heir health shown twice on examine